controllers/storagecluster/external_ocs.go

```diff
@@ -107,9 +107,20 @@ func (r *StorageClusterReconciler) acknowledgeOnboarding(instance *ocsv1.Storage
 	instance.Status.Phase = statusutil.PhaseProgressing
 
 	r.Log.Info("External-OCS:Onboarding is acknowledged successfully.")
+
 	return reconcile.Result{Requeue: true}, nil
 }
 
+func (r *StorageClusterReconciler) reconcileConsumerStatusReporterJob(instance *ocsv1.StorageCluster, externalClusterClient *externalClient.OCSProviderClient) (reconcile.Result, error) {
```

Comment:
IMO we are not required this extra wrapper func as we are just calling one func here which can be called from the calling site itself.

Reply:
Updated in latest commit